### ISSUE TITLE
default SMS source: sms_carriers option

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#3877]
`sms_carriers` option should include **all** the SMS carriers in DB.

It looks like the latest additions in DB haven't been added in `conf/authentication.conf`

**Comment by julsemaan** [comment]
Yeah this is because we reference the SMS carriers in authentication.conf as well.

We have 2 choices:

 1. We adjust authentication.conf.example
 1. We support making the sms_carriers empty and when its the case, it displays all the carriers in the DB

Would be cleaner to do option 2 and it would be future-proof

**Comment by nqb** [comment]
In option 2, admin can select SMS carriers in a dropdown list in the auth source. 

However, if they don't, the default sms source will not work: guest can't select a provider which has not been defined in the source.

**Comment by julsemaan** [comment]
This is why we need to change the backend code to magically fill in all the carriers when the list is empty

**Comment by extrafu** [comment]
@nqb Please add manually the ones that are missing and open a PR.

**Comment by nqb** [comment]
For 8.3, I fixed missing sms_carriers in 7a1af6c
